sysl2/sysl/cmd_runner.go

```diff
@@ -24,6 +33,11 @@ func (r *cmdRunner) Run(which string, fs afero.Fs, logger *logrus.Logger) error
 			var mod *sysl.Module
 			var err error
 			if cmd.RequireSyslModule() {
+				err = r.getProjectRoot(fs, logger)
```

Comment:
nitpick: when a func only returns an error, put it in a single line ```if err := func(); 
err != nil { 
return err 
}```

Reply:
changed to that